quartz/components/ContentMeta.tsx

```diff
@@ -23,22 +26,28 @@ export default ((opts?: Partial<ContentMetaOptions>) => {
     const text = fileData.text
 
     if (text) {
-      const segments: string[] = []
+      const segments: (string | JSX.Element)[] = [];
 
       if (fileData.dates) {
         segments.push(formatDate(getDate(cfg, fileData)!, cfg.locale))
       }
 
       // Display reading time if enabled
       if (options.showReadingTime) {
+        if(options.showComma){
+          segments.push(",")
+        }
```

Comment:
this should be done at the `join` level, not just as a segment

each segment should be a standalone info block (e.g. reading time, status, etc.)

Reply:
Not sure I follow since by shifting to using `<span>` elements the join was done away with. Are you saying that within the standalone segment I should do a `join` to add in a comma where needed before pushing the segment to segments?

Reply:
since we wrap each components with a span, you can join the the list with comma altogether later.

I think that is what jacky meant.

Reply:
gotcha, I think this is fixed now